src/templates/profile.ts

```diff
@@ -2,16 +2,67 @@
  * Returns a usecase header with filled in `name` and `version`.
  */
 export function header(name: string, version: string): string {
-  return `name = "${name}"
+  return `"""
+${name} Profile
```

Comment:
I would personally avoid _“Profile”_ keyword as much as possible due to its undefined nature, it's just weird 😄 

What about transforming the original profile name into a proper spaced upper-cased words, e.g. `address/ip-geolocation` would become `Address Ip Geolocation` using [lodash startSpace](https://lodash.com/docs/4.17.15#startCase). Isn't that enough to indicate that should be a human readable title? Remember it also has specific comment-like highlight inside the editor.